visualization.js

```diff
@@ -0,0 +1,141 @@
+      }
+    });
+
+	// load data from csv
+    d3.csv("route.csv", function(error, track) {
+
+		var browser_width = window.innerWidth;
+		// var browser_height = window.innerHeight;
+
+		var initial_offset = window.pageYOffset;
+
+		var width = browser_width,
+			height = 150;
+
+		var projection = d3.geoMercator()
+		    .scale(5*(width + 1) / 2 / Math.PI)
+		    .translate([width / 2, height / 4])
+		    .rotate([60, 23, 0])
+		    .precision(.1);
+
+		var path = d3.geoPath()
+		    .projection(projection);
+
+		var graticule = d3.geoGraticule();
+
+		var svg = d3.select("body").append("svg")
+		    .attr("width", width)
+		    .attr("height", height)
+		    .attr("class", "map");
+
+		svg.append("path")
+		    .datum(graticule)
+		    .attr("class", "graticule")
+		    .attr("d", path);
+
+		var new_track = track.filter(function filter_data(d){ if (d['plaats'] <= 1) { return true; } });
+
+
+		d3.json("world-50m.json", function(error, world) {
+			    
+		    svg.insert("path", ".graticule")
+		      .datum(topojson.feature(world, world.objects.land))
+		      .attr("class", "land")
+		      .attr("d", path);
+
+		    svg.insert("path", ".graticule")
+		      .datum(topojson.mesh(world, world.objects.countries, function(a, b) { return a !== b; }))
+		      .attr("class", "boundary")
+		      .attr("d", path);
+		    
+			
+
+		});
+		
+
+		// draw route
+		var line_generator = d3.line()
+			    .x(function(track) { return projection([track.lat,track.lon])[0] ; })
+			    .y(function(track) { return projection([track.lat,track.lon])[1]; });
+
+		var route_zuid_amerika = svg.append("path")
+			    .attr("d",line_generator(new_track))
+			    .attr("class","route");
+		
+		var update_map = function(event) {
+      		
+
+		    var lowestTriggerAboveThreshold;
+		    var threshold = window.pageYOffset - initial_offset;
+		      
+		    // check which trigger is below the threshold
+		    triggerPositionMap.forEach(function (trigger) {
+		        if (trigger.top < threshold) {
+		          if (!lowestTriggerAboveThreshold || trigger.top > lowestTriggerAboveThreshold.top) {
+		            lowestTriggerAboveThreshold = trigger;
+		        	}
+		        }
+		    });
+
+
+			var new_track = track.filter(function filter_data(d){ if (d['plaats'] <= parseInt(lowestTriggerAboveThreshold.id)) { return true; } });
+		    var route = d3.select(".route");
+			
+			route.remove();
+			
+			var route_zuid_amerika = svg.append("path")
+			    .attr("d",line_generator(new_track))
+			    .attr("class","route");
+		    
+
+		    console.log(lowestTriggerAboveThreshold.id);
+		    // var dataToUse = dashboardData[0];
+
+		    // if (lowestTriggerAboveThreshold) {
+
+		    //     dataToUse = dashboardData[lowestTriggerAboveThreshold.id];
+		    //     kosten_value.innerHTML = dataToUse['money'];
+		    //     documenten_value.innerHTML = dataToUse['documents'];
+		    //     displayEmoticon(dataToUse['frustratie']);
+	        
+	     //  	}
```

Comment:
Generally, don't leave any commented out or unused code. Since we're merging this into master later, it's good to keep things clean and purposeful.